torch/_higher_order_ops/effects.py

```diff
@@ -6,6 +6,7 @@
 import torch
 import torch.utils._pytree as pytree
 from torch._C import DispatchKey
+from torch._higher_order_ops.print import print
```

Comment:
can you do `from torch._higher_order_ops.print import print as hoo_print` or something, so that if someone ever has to actually `print` something while debugging it doesnt conflict with the HOO

Reply:
great point! Fixed!